components/homepage/skills/SkillCard.tsx

```diff
@@ -0,0 +1,24 @@
+import Image from 'next/image';
+
+import { SkillsType } from '@/types';
+import MotionWrapperCardDiv from '@/HOC/MotionWrapperCardDiv';
+
+const SkillCard = ({ _id, image, altText, index }: SkillsType) => (
```

Comment:
You're in fact doing that here, not using a return statement and brackets. Whichever direction you choose, keep consistency.